diff -durN wmget.orig/Makefile wmget/Makefile
--- wmget.orig/Makefile	Tue Jul  2 13:24:11 2002
+++ wmget/Makefile	Tue Jul  2 13:40:27 2002
@@ -52,7 +52,7 @@
 INSTALLBIN=	install -m 555
 INSTALLMAN=	install -m 444
 CC=		gcc
-CFLAGS=		-ansi -Wall -W -I/usr/X11R6/include -D_GNU_SOURCE
+CFLAGS=		-ansi -Wall -W -I/usr/X11R6/include -D_GNU_SOURCE -O2 -march=i686
 # The following line is what I use during development
 #CFLAGS:=	$(CFLAGS) -Werror -g
 LDFLAGS=	-L/usr/X11R6/lib -lXpm -lXext -lX11 -lm -lcurl
diff -durN wmget.orig/Makefile~ wmget/Makefile~
--- wmget.orig/Makefile~	Thu Jan  1 01:00:00 1970
+++ wmget/Makefile~	Tue Jul  2 13:37:37 2002
@@ -0,0 +1,193 @@
+# Copyright (c) 2001-2002 Aaron Trickey <aaron@amtrickey.net>
+# 
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+# 
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+# 
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+# AUTHOR BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN
+# AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+# Makefile for the ``wmget'' (formerly ``wmcurl'') project.
+# This Makefile requires GNU make and probably other GNU stuff...
+
+
+########################################################################
+# Build Targets:
+#
+# all [default]:	Builds the wmget application and documentation
+# install:		Installs the application
+# uninstall:		Attempts to uninstall, if this makefile installed it
+# dockapplib:		Recurses into the dockapp dir and builds the library
+# doc:			Builds all documentation
+# depend:		Runs makedepend, regenerating Makefile deps
+# clean:		Cleans, except for generated HTML/man docs
+# docclean:		Cleans generated docs
+# slackpkg:		Builds a Slackware package in packages/slackware
+# sourceball:		Builds a source+docs tarball in packages/source
+########################################################################
+
+all:		wmget doc
+
+.PHONY:		all install uninstall dockapplib doc depend clean \
+		docclean slackpkg sourceball
+
+
+##### BUILD SETTINGS AND VARIABLES #####################################
+
+# To specify a different prefix, you can override this on the command line
+#	make PREFIX=/opt/dockapps install
+PREFIX=		/usr/local
+
+INSTALLDIR=	install -d -m 755
+INSTALLBIN=	install -m 555
+INSTALLMAN=	install -m 444
+CC=		gcc
+CFLAGS=		-ansi -Wall -W -I/usr/X11R6/include -D_GNU_SOURCE "$OPTFLAGS"
+# The following line is what I use during development
+#CFLAGS:=	$(CFLAGS) -Werror -g
+LDFLAGS=	-L/usr/X11R6/lib -lXpm -lXext -lX11 -lm -lcurl
+DOCS=		wmget.1
+
+VERSION:=	$(shell grep '\#define WMGET_VERSION ' wmget.h \
+			| sed -e 's/.*"\(.*\)".*/\1/' )
+
+OBJS=		server.o \
+		request.o \
+		cancel.o \
+		list.o \
+		retrieve.o \
+		iq.o \
+		wmget.o
+
+DALIBDIR=	dockapp
+DALIB=		$(DALIBDIR)/dockapp.a
+
+ALL_SRCS=	$(subst .o,.c,$(OBJS))
+
+
+##### PROGRAM ##########################################################
+
+install:	all
+		@if [ -f /etc/slackware-version ] ; then \
+			echo '***' You seem to be running Slackware... ; \
+			echo Run \"make slackpkg\" to build an installable \
+			     package in packages/slackware ; \
+		else \
+			echo $(PREFIX) > install.prefix ; \
+			$(INSTALLDIR) $(PREFIX)/bin ; \
+			$(INSTALLBIN) wmget $(PREFIX)/bin/wmget ; \
+			$(INSTALLDIR) $(PREFIX)/man/man1 ; \
+			$(INSTALLMAN) wmget.1 $(PREFIX)/man/man1/wmget.1 ; \
+		fi
+
+uninstall:
+		cd `cat install.prefix` && rm -f bin/wmget man/man1/wmget.1
+		rm -f install.prefix
+
+wmget:		dockapplib $(OBJS)
+		$(CC) $(CFLAGS) $(LDFLAGS) $(OBJS) $(DALIB) -o $@
+
+depend:		
+		makedepend -- $(CFLAGS) -- $(ALL_SRCS)
+
+
+##### LIBRARY ##########################################################
+
+dockapplib:
+		make -C $(DALIBDIR)
+
+
+
+##### DOCUMENTATION ####################################################
+
+doc:		$(DOCS)
+
+wmget.1:	wmget.pod
+		pod2man --section=1 --release=' ' \
+			--center='User Manuals' $< > $@
+
+wmget.html:	wmget.pod
+		pod2html --infile=$< --outfile=$@
+
+
+##### CLEANUP ##########################################################
+
+clean:
+		rm -f *.o wmget core pod2html-* Makefile.bak install.prefix \
+			wmget.html
+		sed -e '/^# DO NOT/q' < Makefile > Makefile.clean \
+		&& mv -f Makefile.clean Makefile
+		make -C dockapp clean
+
+docclean:
+		rm -f $(DOCS)
+
+
+##### SLACKWARE PACKAGE ################################################
+
+SLACK_PFX=	slackroot/$(PREFIX)
+SLACK_PACKAGE=	packages/slackware/wmget-$(VERSION).tgz
+
+slackpkg:	wmget doc
+		-mkdir -p packages/slackware
+		rm -rf slackroot
+		$(INSTALLDIR) $(SLACK_PFX)
+		$(INSTALLDIR) $(SLACK_PFX)/bin
+		$(INSTALLDIR) $(SLACK_PFX)/man
+		$(INSTALLDIR) $(SLACK_PFX)/man/man1
+		$(INSTALLBIN) wmget $(SLACK_PFX)/bin/wmget
+		$(INSTALLMAN) wmget.1 $(SLACK_PFX)/man/man1/wmget.1
+		cd slackroot && \
+		tar czv	-f ../$(SLACK_PACKAGE) 	\
+			--owner=root 		\
+			--group=root 		\
+			*
+		rm -rf slackroot
+
+
+##### SOURCE PACKAGE ###################################################
+
+SOURCEBALL=	packages/source/wmget-$(VERSION)-src.tar.gz
+
+sourceball:	doc clean
+		-mkdir -p packages/source
+		cd .. &&			\
+		tar czv	--exclude RCS 		\
+			--exclude .\*		\
+			--exclude packages	\
+			--exclude tags		\
+			--exclude \*~		\
+			--exclude working	\
+			--exclude www		\
+			-f wmget/$(SOURCEBALL) 	\
+			wmget
+
+
+##### WEB SITE #########################################################
+
+WWW_ROOT=	$(HOME)/amtrickey.net/public_html
+WWW_DOWNLOAD=	$(WWW_ROOT)/download
+WWW_WMGET=	$(WWW_ROOT)/wmget
+
+www:		slackpkg sourceball wmget.html
+		cp $(SLACK_PACKAGE) $(WWW_DOWNLOAD)
+		cp $(SOURCEBALL) $(WWW_DOWNLOAD)
+		cp README NEWS wmget.html $(WWW_WMGET)
+
+
+
+# END of human-written part!  All the dependencies below are generated
+# by `makedepend', invoked via `make depend'.  It scans for the DO NOT DELETE
+# line you see below and replaces everything after it.
+
+# DO NOT DELETE THIS LINE -- make depend depends on it.
Binary files wmget.orig/dockapp/da_mouse.o and wmget/dockapp/da_mouse.o differ
Binary files wmget.orig/dockapp/da_run.o and wmget/dockapp/da_run.o differ
Binary files wmget.orig/dockapp/da_x.o and wmget/dockapp/da_x.o differ
Binary files wmget.orig/dockapp/dockapp.a and wmget/dockapp/dockapp.a differ
