diff -urN wmget.orig/dockapp/Makefile wmget/dockapp/Makefile
--- wmget.orig/dockapp/Makefile	2004-04-20 19:44:39.805916032 +0200
+++ wmget/dockapp/Makefile	2004-04-20 19:47:40.433456472 +0200
@@ -22,7 +22,7 @@
 
 DAOBJS=		da_run.o da_x.o da_mouse.o
 
-CFLAGS=		-ansi -Wall -W -Werror -g
+CFLAGS=		-ansi -Wall -W -Werror $(OPTFLAGS)
 LDFLAGS=	-L/usr/X11R6/lib -lXpm -lXext -lX11
 
 dockapp.a:	$(DAOBJS)
diff -urN wmget.orig/Makefile wmget/Makefile
--- wmget.orig/Makefile	2004-04-20 19:44:39.804916184 +0200
+++ wmget/Makefile	2004-04-20 19:46:53.566581320 +0200
@@ -51,7 +51,7 @@
 INSTALLBIN=	install -m 555
 INSTALLMAN=	install -m 444
 CC=		gcc
-CFLAGS=		-Wall -W -I/usr/X11R6/include -O
+CFLAGS=		-Wall -W -I/usr/X11R6/include $(OPTFLAGS)
 # The following line is what I use during development
 #CFLAGS:=	$(CFLAGS) -Werror -g 
 LDFLAGS=	-L/usr/X11R6/lib -lXpm -lXext -lX11 -lm -lcurl
