diff -durN wmget.orig/Makefile wmget/Makefile
--- wmget.orig/Makefile	Tue Jul  2 15:34:01 2002
+++ wmget/Makefile	Tue Jul  2 15:40:09 2002
@@ -52,7 +52,7 @@
 INSTALLBIN=	install -m 555
 INSTALLMAN=	install -m 444
 CC=		gcc
-CFLAGS=		-ansi -Wall -W -I/usr/X11R6/include -D_GNU_SOURCE
+CFLAGS=		-ansi -Wall -W -I/usr/X11R6/include -D_GNU_SOURCE $(OPTFLAGS)
 # The following line is what I use during development
 #CFLAGS:=	$(CFLAGS) -Werror -g
 LDFLAGS=	-L/usr/X11R6/lib -lXpm -lXext -lX11 -lm -lcurl
